src/openforms/js/components/admin/form_design/useConfirm.stories.js

```diff
@@ -0,0 +1,37 @@
+import {useState} from 'react';
+
+import ActionButton from '../forms/ActionButton';
+import useConfirm from './useConfirm';
+
+const ButtonWithUseConfirm = () => {
+  const [ConfirmationModal, confirm] = useConfirm(
+    'A sample confirmation message',
+    'The confirmation title'
+  );
+  const [confirmationResult, setConfirmationResult] = useState(null);
+  return (
+    <div>
+      <ActionButton
+        text="Open confirmation modal"
+        onClick={async () => {
+          const result = await confirm();
+          setConfirmationResult(result);
+        }}
+      />
+      {confirmationResult !== null ? (
+        <p>Confirmation result: {confirmationResult.toString()}</p>
+      ) : null}
+      <ConfirmationModal />
+    </div>
+  );
+};
+
+export default {
+  title: 'Admin / Custom / UseConfirm',
+  render: () => <ButtonWithUseConfirm />,
+  component: useConfirm,
```

Comment:
It might be good to add a basic test here to open the modal and check that it has the expected text

Reply:
Ah yeah check, will do!